daemon/cmd/daemon_main.go

```diff
@@ -1607,14 +1605,12 @@ func initKubeProxyReplacementOptions() {
 
 	if option.Config.EnableNodePort {
 		if option.Config.EnableIPSec {
-			msg := "IPSec cannot be used with NodePort BPF."
+			msg := "IPSec cannot be used with BPF NodePort."
 			if strict {
 				log.Fatal(msg)
 			} else {
-				option.Config.EnableHostPort = false
-				option.Config.EnableNodePort = false
-				option.Config.EnableExternalIPs = false
-				log.Warn(msg + " Disabling the feature.")
+				disableNodePort()
+				log.Warn(msg + " Disabling BPF NodePort feature.")
```

Comment:
:+1: for the message change. It always takes me a minute to remember which feature is disabled :-)